Gemfile.lock

```diff
@@ -214,6 +214,9 @@ GEM
     parser (3.0.2.0)
       ast (~> 2.4.1)
     pg (1.2.3)
+    pg_search (2.3.6)
```

Comment:
we should run `BUNDLE_GEMFILE=Gemfile.next bundle install` too so it updates the Gemfile.next.lock with this new dependency